package.json

```diff
@@ -97,7 +97,8 @@
     "ts-node": "^7.0.0",
     "tsconfig-paths": "^3.1.1",
     "typescript": "^2.4.2",
-    "url-parse": "^1.2.0"
+    "url-parse": "^1.2.0",
+    "nodelist-foreach-polyfill": "^1.2.0"
```

Comment:
List of packages should be in alphabetical order. Please move into right position.

Reply:
Well spotted, addressed.